src/material-experimental/mdc-chips/chip-grid.ts

```diff
@@ -337,6 +335,10 @@ export class MatChipGrid
   /** Associates an HTML input element with this chip grid. */
   registerInput(inputElement: MatChipTextControl): void {
     this._chipInput = inputElement;
+
+    if (this._ariaDescribedby) {
+      this._chipInput.setDescribedByIds(this._ariaDescribedby.split(' '));
```

Comment:
Instead of having to split the IDs here, maybe we should just save the string array?

Reply:
This field comes from `MatChipSet`, which `MatChipGridBase` extends:
https://github.com/angular/components/blob/f05e65a02b636d7c3d4e181f65ac0d7e94e091fd/src/material-experimental/mdc-chips/chip-set.ts#L142

The field is defined as a `string`.  I'm more than okay to save the values in an array, but I suspect we'll want to keep the value of `_ariaDescribedby` up to date regardless.

If we're comfortable introducing a new field to store the array, I'm more than happy to do so; let me know one way or the other.

Reply:
Hmm, I think that the `aria-describedby` on `MatChipSet` isn't doing much since the element isn't focusable. It should be moved to the chip listbox.

Reply:
Removed.